app/renderer/components/lobby/modals/SessionSettings.css

```diff
@@ -8,3 +8,8 @@
 .sessionMode button:not(:first-child) {
   margin-left: 1rem;
 }
+
+select.maxUsers {
+    mix-blend-mode: unset;
+    margin-bottom: 1rem;
+}
```

Comment:
Install the [Prettier](https://prettier.io/) extension in your preferred editor and it should autoformat this according to the project settings.

Reply:
`select` can be dropped from the selector as well. This project uses [CSS modules](https://github.com/css-modules/css-modules) so there isn't any worry of class name collisions.

Reply:
I added the tag selector to take priority above the default dropdown style. It's better than using id or !important, i guess.